worlds/ladx/__init__.py

```diff
@@ -534,12 +534,12 @@ def fill_slot_data(self):
             ]
 
             # use the default behaviour to grab options
-            slot_data = self.options.as_dict(*slot_options)
+            slot_data.update(self.options.as_dict(*slot_options))
 
             # for options which should not get the internal int value but the display name use the extra handling
             slot_data.update({
                 option: value.current_key
                 for option, value in dataclasses.asdict(self.options).items() if option in slot_options_display_name
             })
 
-        return slot_data
+        return slot_data
```

Comment:
Accidentally removed an EoF newline
```suggestion
        return slot_data

```